esmvaltool/preprocessor/_regrid.py

```diff
@@ -141,6 +154,10 @@ def regrid(cube, target_grid, scheme):
         latitude (degrees) for a global, regular target grid.
     scheme : str
         The regridding scheme to perform, see `regrid.HORIZONTAL_SCHEMES`.
+    lat_offset : bool
+        Offset the grid center on LAT coordinate. Default True.
```

Comment:
Be a bit more informative, e.g.
```suggestion
        Offset the grid centers of the latitude coordinate w.r.t. the pole by half a grid step.
```
and similar for `lon_offset`. There is no need to mention the default setting, as this information is included automatically.

Reply:
good call! - changed in both functions